Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -57,6 +57,7 @@ def tests
       ENV["SEED"] = ARGV.next if ARGV.include? "--seed"
 
       files = Dir.glob("test/**/*_{spec,test}.rb")
+                 .reject { |p| p =~ %r{^test/vendor} }
```

Comment:
Make this `test/vendor/bundle`, since there is at least one test actually testing vendored code (`plist`, currently still in `cask-tests`).